helm/athena/templates/podlogs.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if and .Values.monitoring .Values.monitoring.podLogs .Values.monitoring.podLogs.enabled }}
+apiVersion: monitoring.grafana.com/v1alpha2
+kind: PodLogs
+metadata:
+  namespace: {{ .Release.Namespace }}
+  name: {{ .Values.name }}
+  labels:
+    {{- include "athena.labels.common" . | nindent 4 }}
+  annotations:
+    giantswarm.io/notes: "Log collection for Athena metadata service"
```

Comment:
I wonder if this actually does anything

Reply:
It doesn't. I didn't even know it existed until I saw it on the RBAC template resources

Reply:
Nah, maybe we'll use it some day